examples/src/main/scala/org/apache/spark/examples/streaming/ActorWordCount.scala

```diff
@@ -39,7 +39,7 @@ case class UnsubscribeReceiver(receiverActor: ActorRef)
 class FeederActor extends Actor {
 
   val rand = new Random()
-  var receivers: LinkedList[ActorRef] = new LinkedList[ActorRef]()
+  var receivers: ArrayBuffer[ActorRef] = new ArrayBuffer[ActorRef]()
```

Comment:
This could just be `var receivers =ArrayBuffer[ActorRef]()`. In fact it can be a `val`; the one other assignment in the line with `dropWhile` is pointless for a mutable collection.

I suppose one could argue we should be using some form of linked list with O(1) insertion at the head, but given it's an example, I'm not so worried.